source/proximity_list/tests/test.cpp

```diff
@@ -368,3 +368,88 @@ BOOST_AUTO_TEST_CASE(test_poi_filter) {
     BOOST_CHECK(poi_names.find("bob") != poi_names.end());
     BOOST_CHECK(poi_names.find("bobette") != poi_names.end());
 }
+
+BOOST_AUTO_TEST_CASE(test_find_stop_points_nearby_options) {
+    navitia::type::Data data;
+
+    // origin
+    navitia::type::GeographicalCoord c;
+    c.set_lon(-1.554514);
+    c.set_lat(47.218515);
+
+    // POIs
+    {
+        auto poitype = new navitia::georef::POIType();
```

Comment:
I know this was how other tests were written, I still prefer not seeing `new` in c++14 :)